iphone/Classes/TiUIiOSStepper.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_UIIOSSTEPPER
+#import "TiUIView.h"
+
+@interface TiUIiOSStepper : TiUIView {
+@private
+    UIStepper *stepper;
+    UIImage *backgroundImageCache;
```

Comment:
So here are my thoughts on this: Since we subclass the `TiUIView` in every UI element, I think we could do all the backgroundImage stuff on that view instead of adding another view that slows down the UI creation. Doing that, we wouldn't even need all the custom setter and logic to add the image, because the `TiUIView` already does all that by default. 

Please test this and verify it's necessary to do all this.